src/editor/StyleEditor.cpp

```diff
@@ -17,7 +17,14 @@
 #include "Settings.h"
 
 namespace StyleEditor {
-const char *SYSTEM_STYLE = "<System>";
+const static char *SYSTEM_STYLE = "<System>";
+static QString currentStyleBaseDir;
+static QString currentStyleName = SYSTEM_STYLE;
+
+void setSystemStyle() {
```

Comment:
what's this fn for? is it still necessary?

Reply:
it sets the theme name var to "<System>" so the new palette functions know when not to try and read from palette.ini
you could probably get rid of it and make the palette functions only check if the string is empty but that seems a little less clean/reliable to me

Reply:
actually i skimmed over the quoted code and after reading it again it's not what i thought it was
currentStyleName and currentStyleBaseDir are variables that are theoretically always set and usable for the palette functions (since they're not guaranteed to actually have these things when they're called)

in contrast the ones in StyleEditor called for other things are guaranteed to have a style name and directory so it's fine for those to be in the call
in this case it's not

Reply:
er sorry I'm just saying, I don't see this function used anywhere. am i missing something?

Reply:
if it's not used then i forgot to remove it